dev/README_RELEASE_PROVIDER_PACKAGES.md

```diff
@@ -392,10 +392,7 @@ breeze release-management publish-docs \
     --package-filter 'apache-airflow-providers-*' \
     --override-versioned
 
-cd "${AIRFLOW_SITE_DIRECTORY}"
-breeze release-management add-back-references --airflow-site-directory --gen-type providers
-cd ..
-
+breeze release-management add-back-references --gen-type providers
```

Comment:
@aminghadersohi  I suspect this doesn't work as expected for redirects between providers like was done in https://github.com/apache/airflow/pull/32306

This caused creation of latest folder in the Google provider:

<img width="296" alt="Screenshot 2023-07-30 at 10 13 15" src="https://github.com/apache/airflow/assets/45845474/55bed2b0-a532-4875-971e-3a669c585ff4">

Which caused failure in doc site build https://github.com/apache/airflow-site/actions/runs/5704523295/job/15458213062

When I removed the latest folder build ran successfully
https://github.com/apache/airflow-site/pull/840/commits/f8b14602eb9c793926fdfd280b876ea6bbd86dec

Reply:
Ah NVM this is issue in the path https://github.com/apache/airflow/pull/32945
I guess we can work on a pre commit to avoid such cases in the future